.golangci.yml

```diff
@@ -1,22 +1,10 @@
 ---
+version: '2'
 run:
-    # GolangCI will not check files with build tags, unless specified here.
     build-tags:
         - integration
-# All rules here were written in the time of GolangCI lint v1.54.1
 linters:
     enable:
-        # By Default
-        - errcheck
-        - gosimple
-        - govet
-        - ineffassign
-        - staticcheck
-        - stylecheck
-        - typecheck # Not in the linter list, but keep it https://golangci-lint.run/usage/faq/#why-do-you-have-typecheck-errors
-        - unused
-
-        # Extra
```

Comment:
Issue: Why you have removed all those? I don't see they are added later.

Reply:
yes I just upgraded. I will redo it from the former version